editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1106,6 +1107,53 @@ void CanvasItemEditor::_switch_theme_preview(int p_mode) {
 	EditorNode::get_singleton()->update_preview_themes(theme_preview);
 }
 
+void CanvasItemEditor::_prepare_translation_menu() {
+	const String current_preview_locale = EditorNode::get_singleton()->get_preview_locale();
+
+	translation_menu->clear();
+	translation_menu->reset_size();
+
+	translation_menu->add_radio_check_item(TTRC("None"));
+	translation_menu->set_item_metadata(-1, "");
+	if (current_preview_locale.is_empty()) {
+		translation_menu->set_item_checked(-1, true);
+	}
+
+	const Vector<String> locales = TranslationServer::get_singleton()->get_loaded_locales();
+	if (!locales.is_empty()) {
+		translation_menu->add_separator();
+	}
+	for (const String &locale : locales) {
+		const String name = TranslationServer::get_singleton()->get_locale_name(locale);
+		translation_menu->add_radio_check_item(name == locale ? name : name + " [" + locale + "]");
```

Comment:
Are locale names translatable?
I think they exist in doc translations at least, not sure about editor's 🤔